src/ui/common/src/components/pages/AccountPage.tsx

```diff
@@ -32,6 +32,55 @@ type AccountPageProps = {
   Layout?: React.FC<LayoutProps>;
 };
 
+/*
```

Comment:
Remove comment.

Need to check storageConfigType and adjust to gcs or s3 accordingly. They have different parameters like buckets and regions. Coming up in a future commit!